adafruit_ads1x15/adafruit_ads1x15.py

```diff
@@ -131,6 +130,23 @@ def __init__(self, i2c, address=ADS1X15_DEFAULT_ADDRESS):
                           ADC_Channel(self, 2),
                           ADC_Channel(self, 3)]
 
+    def _write_register(self, reg, value):
+        """Write 16 bit value to register."""
+        self.buf[0] = reg
+        self.buf[1] = (value >> 8) & 0xFF
+        self.buf[2] = value & 0xFF
+        with self.i2c_device as i2c:
+            i2c.write(self.buf)
```

Comment:
Any reason for not using Adafruit_CircuitPython_Register?

Reply:
Just haven't used it yet. I'll take a look.

Reply:
Deferring this to a future PR.